frontman.go

```diff
@@ -0,0 +1,80 @@
+package frontman
+
+import (
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"runtime"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type Frontman struct {
+	Config *Config
+
+	// internal use
+	httpTransport *http.Transport
+	hubHTTPClient *http.Client
+	hostInfoSent  bool
+
+	offlineResultsBuffer []Result
+
+	rootCAs *x509.CertPool
+	version string
+}
+
+func New(version string, cfg *Config) *Frontman {
```

Comment:
Can you please switch the parameters? Then we're consistent with `cagent`.

Reply:
done